braindecode/models/sleep_stager_eldele_2021.py

```diff
@@ -167,7 +173,7 @@ def forward(self, x):
 
 
 class _MRCNN(nn.Module):
-    def __init__(self, after_reduced_cnn_size):
+    def __init__(self, after_reduced_cnn_size, shhs):
```

Comment:
I would not hard code anything for a specific dataset. Just check the value of sfreq? ok for you?